src/transformers/models/whisper/tokenization_whisper.py

```diff
@@ -89,9 +89,130 @@ def get_pairs(word):
     return pairs
 
 
+LANGUAGES = {
```

Comment:
Currently using `language` (e.g. `spanish`) instead of `lang_id` (`es`) -> this is how the original Whisper model does it. If there's a preference for `lang_id` I'm happy to switch!